app/api/helpers/system_mails.py

```diff
@@ -209,11 +209,11 @@
     },
     MAIL_TO_EXPIRED_ORDERS: {
         'recipient': 'User',
-        'subject': u'Tickets for {event_name} are still available ',
+        'subject': u'Your order for {event_name} has expired. Visit to order a ticket',
```

Comment:
@uds5501 `Visit to order a ticket` its meaning is unclear, can it be more meaningful?

Reply:
@kushthedude fixed it